models/Item.js

```diff
@@ -0,0 +1,55 @@
+// import important parts of sequelize library
+const { Model, DataTypes } = require('sequelize');
+// import our database connection from config.js
+const sequelize = require('../config/connection');
+
+// Initialize Item model (table) by extending off Sequelize's Model class
+class Item extends Model {}
+
+// set up fields and rules for Item model
+Item.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+      allowNull: false
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    description: {
+      type: DataTypes.STRING,
```

Comment:
This needs to be type: DataTypes.TEXT, it allows for more characters. .STRING is only 225 characters. Schema mockup has been updated to reflect this.